docs/data-companion/grpc.md

```diff
@@ -0,0 +1,219 @@
+---
+order: 1
+parent:
+    title: gRPC services
+    order: 2
+---
+
+
+# Fetching data from the node
+
+One of the most important steps to create a Data Companion service is to extract the necessary data from the node.
+Fortunately, CometBFT provides gRPC endpoints that allow you to fetch the data, such as `version`, `block` and
+`block results`.
+
+This documentation aims to provide a detailed explanation of CometBFT's gRPC services that can be used to retrieve
+the data you need.
+
+## Enabling the gRPC services
+
+To utilize the gRPC services, it's necessary to enable them in CometBFT's configuration settings.
+
+In the `[gRPC]` section of the configuration:
+```
+#######################################################
+###       gRPC Server Configuration Options         ###
+#######################################################
+
+#
+# Note that the gRPC server is exposed unauthenticated. It is critical that
+# this server not be exposed directly to the public internet. If this service
+# must be accessed via the public internet, please ensure that appropriate
+# precautions are taken (e.g. fronting with a reverse proxy like nginx with TLS
+# termination and authentication, using DDoS protection services like
+# CloudFlare, etc.).
+#
+
+[grpc]
+```
+
+Add the address for the non-privileged (regular) services, for example:
+
+```
+# TCP or UNIX socket address for the RPC server to listen on. If not specified,
+# the gRPC server will be disabled.
+laddr = "tcp://0.0.0.0:26090"
+```
+
```

Comment:
The note from line 111 should come here, actually.

Reply:
Done.